rush/rush-lib/src/RushConstants.ts

```diff
@@ -6,32 +6,45 @@
  *
  * @public
  */
-export class RushConstants {
+export namespace RushConstants {
```

Comment:
@iclanton **api-extractor** supports now namespaces that contain a table of simple constants.  You still need to use classes-with-static-members for functions and property-getters.

---
In reply to: [115868699](https://github.com/Microsoft/web-build-tools/pull/176#discussion_r115868699) [](ancestors = 115868699)